chain/network/src/types.rs

```diff
@@ -506,6 +506,7 @@ impl Message for GetPeerId {
 #[derive(MessageResponse, Debug)]
 #[cfg_attr(feature = "test_features", derive(serde::Serialize))]
 pub struct GetPeerIdResult {
+    #[allow(unused)]
```

Comment:
Seeing these I often ask myself "why?". Could you document all the new instances of `allow(unused)`?

Reply:
@frol Rust, since `1.57.0` started detecting unused values with `pub(crate)` modifier.

Reply:
Do we know why we still need that field to be `pub(crate)` or even exist at all?

Reply:
@frol Yes, it's a test only feature. Take a look at https://github.com/near/nearcore/pull/5663